recipes/qiskit-aer/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set name = "qiskit-aer" %}
+{% set version = "0.11.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Qiskit/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: d18aefd09971bfe2018dc78623ddf8a25bc0660f4fb0987a82c8e2a677bffc4e
+  patches:
+    # This patch is used to prevent scikit-build from trying to pip install cmake.
+    # It can be dropped when a version including
+    # https://github.com/Qiskit/qiskit-aer/pull/1693 is released, likely
+    # version 0.12.0.
+    - 0001-Drop-setup_requires.patch
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  # Skip Windows build until qiskit-terra has a Windows build
+  # Skip macOS build until https://github.com/Qiskit/qiskit-aer/issues/1689 is addressed
+  skip: True  # [win or osx]
+  number: 0
+  script_env:
+    - DISABLE_DEPENDENCY_INSTALL=ON
+    - DISABLE_CONAN=ON
+    - SKBUILD_CONFIGURE_OPTIONS="-GNinja"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake !=3.17,>=3.8
+    - libgomp      # [linux]
+    - llvm-openmp  # [osx]
+    - ninja
+  host:
+    - libblas
+    - liblapack
+    - muparserx =4.0.8
+    - nlohmann_json =3.1.2
+    - numpy
+    - pip
+    - python
+    - pybind11
+    - scikit-build >=0.11.0
+    - setuptools >=40.1.0
+    - spdlog =1.5.0
```

Comment:
Q: Is this necessary? 1.5.0 is quite old.

Reply:
Likely not. I took it from [qiskit-aer's cmake configuration](https://github.com/Qiskit/qiskit-aer/blob/6b0e9996e907b48a3788b15ac2010aa8e1f32ff7/cmake/dependency_utils.cmake#L27). Since then it was bumped to 1.9.2 in https://github.com/Qiskit/qiskit-aer/pull/1637 without comment, so I assume the version bump was incidental and not related to the other code changes.

I am not sure what the best practice is here for conda. Since spdlog is a header only project, I was assuming it would only influence the aer binary and not the rest of the conda environment so it was best to mirror the upstream pin, but we could possibly use just set a minimum version?